project/backend/src/main/java/com/google/sps/data/RestaurantScorer.java

```diff
@@ -6,6 +6,7 @@
 public final class RestaurantScorer {
   private static final int MAX_RATING = 5;
   private static final int MID_RATING = 3;
+  private static final int NUM_INITIAL_RATINGS = 20;
```

Comment:
add a comment here explaining the purpose of this var (& why its made up rn) - basically what you wrote in the PR description